27-11-208/greg/bowling-game/game.rb

```diff
@@ -0,0 +1,54 @@
+require 'byebug'
+
+class Game
+
+  def initialize
+    @frames = [[]]
+  end
+
+  attr_accessor :frames
+
+  def roll(pins)
+    frames.size <= 9 ? handle_rolls(pins) : frames.last << pins
+  end
+
+  def score
+    scores = frames.map.with_index do |frame, index|
```

Comment:
Could use frames.reduce instead of frames.map and then calling reduce afterwards